### DISCUSSION TITLE
placeholders replaced but brackets remain

### DISCUSSION THREAD

**Discussion by nmaylevc** [CMSgov/hospital-price-transparency#128] [Q&A] [answered]
Please clarify how the substitution should be done for [payer_name] and other placeholders.  We see hospital price transparency files with the brackets include in the file surrounding the substituted value.  My understanding was that the [payer_name], [plan_name], and [state] should be replaced by the corresponding item and when that happened, the brackets were no longer necessary. The documentation refers to the examples for the specifics on how the substitution should be done.   It seems that the documentation on "Placeholders" is not clear enough and some didn't notice how this worked in the examples.

This is happening in CSV files, but we have also seen this in JSON files.  Here is an actual example from a recently posted file:

>  {
              "payer_name": "[Cigna]",
              "plan_name": "[NY]",
              "standard_charge_dollar": 474.473,
              "methodology": "percent of total billed charges"
            },

**Comment by RidenourK** [comment, marked as answer]
@nmaylevc. Thank you for your suggestion that CMS clarify in future guidance that hospitals do not need to include the brackets when replacing values in placeholders in the MRF templates.

**Comment by nmaylevc** [comment]
The other thing we have seen is hospitals repeating the min and max fields in the wide CSV format.  It would be helpful to be very explicit as to which fields are repeated (with substitution) per payer-plan and which fields one only needs one of.
